src/builder.rs

```diff
@@ -1519,11 +1554,13 @@ impl<T> SecuritySchemeBuilder<T> {
         self
     }
 
+    /// Sets the key to be used for the security scheme.
```

Comment:
`for referring to the security scheme in the Thing::security and Form::security fields`